apl-desktop/pom.xml

```diff
@@ -222,6 +222,12 @@
       <artifactId>javafx-swing</artifactId>
       <version>${javafx.version}</version>
     </dependency>
+<!--    JavaScript debugger for JavaFX-->
+    <dependency>
+      <groupId>com.vladsch.javafx-webview-debugger</groupId>
+      <artifactId>javafx-webview-debugger</artifactId>
+      <version>0.7.6</version>
```

Comment:
why not the latest version? (0.8.6)

Reply:
It works fine with 0.7.6, but new version '0.8.6' causes npe and does not work properly, also 0.7.6 was fully tested so that's a reason to keep it, also considering the fact that we are moving to cordova/electron desktop